src/MountedTraversal.js

```diff
@@ -6,13 +6,16 @@ import {
   isCompoundSelector,
   AND,
 } from './Utils';
-import {
+import reactCompat from './react-compat';
+const {
   isDOMComponent,
   isCompositeComponent,
   isCompositeComponentWithType,
   isElement,
   findDOMNode,
-} from './react-compat';
+} = reactCompat;
```

Comment:
Why don't we fix this in `react-compat.js`?

Just `export` all these individually.